exodus-installer.sh

```diff
@@ -214,3 +243,5 @@ EOF
 # pass arguments to main function
 #
 exodus_installer $@
+
+# vim: ts=2 sw=2 et
```

Comment:
Please remove it.

Reply:
Please remove what? The `# vim:` line?

Reply:
Yes, config line for vim.

Reply:
I certainly can. But don't you want to enforce uniform indentation? Do you prefer hard tabs? I just like to stick with your standard.

Reply:
I like one style. But not sure that most of people uses Vim... I personally used sublime => vim => sublime and atom in a last few days... instead hardcoding config to each file, better use editorconfig.org

Reply:
Okay, will do my friend!